tests/aws/services/apigateway/test_apigateway_basic.snapshot.json

```diff
@@ -116,5 +116,138 @@
         }
       }
     }
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::TestAPIGateway::test_api_gateway_mock_integration": {
+    "recorded-date": "29-05-2024, 20:06:17",
+    "recorded-content": {
+      "mocked-response": {
+        "echo": "foobar",
+        "response": "mocked"
+      }
+    }
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::test_rest_api_multi_region[path_based_url-GET]": {
+    "recorded-date": "29-05-2024, 23:23:26",
+    "recorded-content": {}
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::test_rest_api_multi_region[path_based_url-ANY]": {
+    "recorded-date": "29-05-2024, 23:24:07",
+    "recorded-content": {}
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::test_rest_api_multi_region[UrlType.HOST_BASED-GET]": {
+    "recorded-date": "29-05-2024, 23:24:41",
+    "recorded-content": {}
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::test_rest_api_multi_region[UrlType.HOST_BASED-ANY]": {
+    "recorded-date": "29-05-2024, 23:25:19",
+    "recorded-content": {}
+  },
```

Comment:
this looks like leftovers

Reply:
Good catch I had intended to use a snapshot, when I realised that asserting 200 was enough for what this test is testing. I will remove.